src/components/container/IntradayHeartRateChart/IntradayHeartRateChart.tsx

```diff
@@ -99,7 +99,16 @@ export default function (props: IntradayHeartRateChartProps) {
         iHrData = undefined;
     }
 
-    const options: MultiSeriesLineChartOptions = { connectNulls: false, hideDots: true, thresholds: props.thresholds, domainMin: yMinValue, domainMax: yMaxValue };
+    const options: MultiSeriesLineChartOptions = {
+        thresholds: props.thresholds,
+        yAxisOptions: {
+            domain: [yMinValue, yMaxValue]
```

Comment:
yMaxValue can be auto following the most recent changes to TimeSeries

Reply:
I'm not sure I understand this completely.  Which recent change are you referring to?

Reply:
No concerns.  Sounds good.